pset-iterators.js

```diff
@@ -138,6 +162,12 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+//  grab list of presidents that have no death year listed
+const livingPresidents = presidents.filter(presidentObj => presidentObj.death_year === null)
+
+//  print to console
+console.log(livingPresidents)
```

Comment:
@sad3xoxo Good job here using the filter method to find all the president's whose death year equal null. 

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L175)